alf/algorithms/ppo_algorithm.py

```diff
@@ -43,6 +43,9 @@ def __init__(self, algorithm: ActorCriticAlgorithm):
 
     def preprocess_experience(self, exp: Experience):
         """Compute advantages and put it into exp.info."""
+        reward = exp.reward
+        if self._algorithm._reward_shaping_fn is not None:
+            reward = self._algorithm._reward_shaping_fn(reward)
         reward = self._algorithm.calc_training_reward(exp.reward, exp.info)
```

Comment:
L49 should be  `reward = self._algorithm.calc_training_reward(reward, exp.info)`

Reply:
Fixed